hazelcast/src/test/java/com/hazelcast/splitbrainprotection/SplitBrainProtectionTest.java

```diff
@@ -53,6 +53,26 @@
 @Category({QuickTest.class, ParallelJVMTest.class})
 public class SplitBrainProtectionTest extends HazelcastTestSupport {
 
+    @Test
+    public void testSplitBrainProtectionNotChecked_whenIsNotEnabled() {
+        String disabledSBPName = "disabled-quorum";
+        SplitBrainProtectionConfig disabledSBPConfig = new SplitBrainProtectionConfig()
+                .setName(disabledSBPName)
+                .setMinimumClusterSize(3);
+
+        MapConfig mapConfig = new MapConfig(randomMapName())
+                .setSplitBrainProtectionName(disabledSBPName);
+
+        Config config = new Config()
+                .addSplitBrainProtectionConfig(disabledSBPConfig)
+                .addMapConfig(mapConfig);
+        HazelcastInstance instance = createHazelcastInstance(config);
+
+        SplitBrainProtectionService quorumService = instance.getSplitBrainProtectionService();
```

Comment:
Uber-minor: naming it `splitBrainProtectionService` or simply not extracting the local variable might be a better alternative since it's not called `QuorumService` anymore.

Reply:
Ah copy-paste from 3.x branch 🤦‍♂